src/components/Header/HeaderLeft.js

```diff
@@ -1,15 +1,14 @@
-import React, { Component } from 'react';
+// import React, { Component } from 'react';
 import './HeaderLeft.css'
+// import HeaderLeft from './HeaderLeft/HeaderLeft';
 
 import HeaderRight from './HeaderRight/HeaderRight';
```

Comment:
<div>
 <HeaderLeft />
 <HeaderRight />
<div/>

Reply:
Calling <HaederRight/> in this Component.